components/Pages/Payruns/SinglePayRun/PayRunItem.js

```diff
@@ -101,11 +132,19 @@ export const PayRunItem = ({
         <HoldPaymentDialog
           invoiceId={invoiceId}
           payRunId={payRunId}
-          isOpen={invoiceId}
           update={updateData}
-          setIsOpened={() => setInvoiceId('')}
+          isOpen={openedModal === 'Hold'}
+          setIsOpened={closeModal}
```

Comment:
setIsOpened=closeModal ?? rename closeModal

Reply:
renamed components prop to closeModal